bitonic-api/src/bitonic_test.zig

```diff
@@ -68,4 +75,133 @@ test "bitonicArray exception case: length greater than possible" {
     };
 
     return;
+}
+
+test "bitonicArray specific case: length 29, start 1, end 15" {
```

Comment:
Nice to have an example of longer sequence but we already covered a test case of a valid expected sequence (bitonicArray specific case: length 7, start 2, end 5), this test becomes redundant